Dockerfile

```diff
@@ -17,4 +16,4 @@ ENV SECURE_PORT=8001
 
 COPY ./target/otp-1.5.0-SNAPSHOT-shaded.jar /otp.jar
 
-CMD java -Xmx"$MEMORY_IN_GB"M -jar /otp.jar --basePath /graphs/$ROUTER_NAME --insecure --inMemory --sharedVehiclesApi $VEHICLES_API --port $PORT --securePort $SECURE_PORT --router $ROUTER_NAME $TRAFFIC_PREDICTION_API
+CMD java -Xmx"$MEMORY_IN_MB"M -jar /otp.jar --basePath /otp --insecure --inMemory --sharedVehiclesApi $VEHICLES_API --port $PORT --securePort $SECURE_PORT --router $ROUTER_NAME $TRAFFIC_PREDICTION_API
```

Comment:
Isn't it necessary to add parameter name for $TRAFFIC_PREDICTION_API?

Reply:
This is terrible design, but we want to add this parameter only for some routers, and in that case parameter name must be a part of this variable. I want to replace it with something like `$EXTRA_PARAMETERS` in the future to avoid confusion.